Assets/Scripts/AI/States/Flee.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+using Random = UnityEngine.Random;
```

Comment:
Is there ambiguity here? I've never needed to do this when calling `Random` because we will never use any other `Random` other than `UnityEngine.Random`.

Reply:
My IDE generates this

Reply:
Which IDE are you using? Are you not on Visual Studio?

Reply:
Using IDEA rider